atomic_reactor/constants.py

```diff
@@ -108,6 +108,8 @@
 PLUGIN_VERIFY_MEDIA_KEY = 'verify_media'
 PLUGIN_EXPORT_OPERATOR_MANIFESTS_KEY = 'export_operator_manifests'
 PLUGIN_PUSH_OPERATOR_MANIFESTS_KEY = 'push_operator_manifests'
+PLUGIN_CONTAINER_SOURCE_BUILD_KEY = 'source_container_build'
```

Comment:
Seeing this alongside the other names I onder if it should be 'build_source_container' (verb at the front).